USAGE.md

```diff
@@ -240,6 +240,15 @@ Above is an example of a constant violation entry in `package_todo.yml`.
 
 Violations exist within the package that makes a violating reference.
 
+
+# Counting Offenses
+
+You can generate a count of all offenses by running:
+
+    bin/packwerk count-offenses
```

Comment:
We should use code blocks like
```
bin/packwerk count-offenses
```
similar to other examples in the docs

It would also be good to add an example of the output.